geocropper/utils.py

```diff
@@ -1837,3 +1837,117 @@ def move_crops_containing_locations(csv_path, source_dir, target_dir):
                             target_dir.mkdir(parents=True, exist_ok=True)
                             shutil.move(str(crop.absolute()), str(target_dir.absolute()))
                             break                 
+
+
+def get_unique_lat_lon(root_dir, speckle):
+    """Loops through every folder of the specified speckle variant in rootdir and returns 
+    a set with unique latitude and longitude positions.
+
+    Parameters
+    ----------
+    root_dir: Path
+        Path of trimmed crops with various speckle variants and recording times.
+    speckle: String
+        Set desired speckle variant (options available in geocropper.stack_trimmed_images) 
+    """
+    
+    # Set is an unordered list which allows no duplicated entries
+    lat_lon_set = set()
+
+    print(f"Reading unique positions for {speckle}...")
+
+    for folder in root_dir.glob(f"*{speckle}*"):
+        if folder.is_dir() and folder.name.split("_")[0] != "stacked":
+            for crop in folder.glob("*"):
+                if crop.is_dir() and crop.name != "0_combined-preview":
+                    lat_lon_set.add(crop.name.split("_")[1] + "_" + crop.name.split("_")[2])
+
+    return lat_lon_set
+
+
+def get_image_path_list(root_dir, speckle, position):
+    """Find one or more images of a specified position in the root directory and returns a list containing the 
+    paths of these images.
+
+    Parameters
+    ----------
+    root_dir: Path
+        Path of trimmed crops with various speckle variants and recording times.
+    speckle: String
+        Set the desired speckle variant (options available in geocropper.stack_trimmed_images)
+    position: String
+        Set the position of the image in latitude and longitude format as a string (e.g. "-68.148781_44.77383")
+    """
+
+    image_path_list = []
+    for folder in root_dir.glob(f"*{speckle}*"):
+        if folder.is_dir() and folder.name.split("_")[0] != "stacked":
+            for crop in folder.glob(f"*{position}*"):
+                if crop.is_dir():
+                    image_path = crop / "sensordata" / "s1_cropped.tif"

```

Comment:
Probably it would be good to also check if s1_cropped.tif exists before appending:
`if image_path.exists():`